Cargo.toml

```diff
@@ -54,33 +54,34 @@ repository = "https://github.com/centrifuge/centrifuge-chain"
 documentation = "https://reference.centrifuge.io/centrifuge_chain/index.html"
 
 [workspace.dependencies]
-hex-literal = { version = "0.3.4" }
+hex-literal = { version = "0.4.1" }
 hex = { version = "0.4.3", default-features = false }
-smallvec = "1.6.1"
-serde = { version = "1.0.119", default-features = false, features = ["derive"] }
-parity-scale-codec = { version = "3.2.2", default-features = false, features = ["derive"] }
-scale-info = { version = "2.3.0", default-features = false, features = ["derive"] }
-log = { version = "0.4", default-features = false }
+smallvec = "1.11.0"
+serde = { version = "1.0.195", default-features = false, features = ["derive"] }
+serde_json = { version = "1.0.111" }
+parity-scale-codec = { version = "3.6.1", default-features = false, features = ["derive"] }
+scale-info = { version = "2.10.0", default-features = false, features = ["derive"] }
+log = { version = "0.4.20", default-features = false }
```

Comment:
Note: As mentioned in the PR description, I have bumped all of these to the versions used in Polkadot SDK v1.7.2